index.html

```diff
@@ -133,7 +138,7 @@ <h3><dfn>Term</dfn> interface</h3>
     <pre class="idl">
     interface Term {
       attribute string termType;
-      attribute string value;
+      attribute (string | Quad) value;
```

Comment:
We have just begun implementation in https://github.com/rdfjs/N3.js/pull/207, where I have noticed that this bothers me. We are violating object homomorphism, which might lead to slowdowns all around. The predictability of the `Term` structure enables high performance; it it can be either of two fundamentally different objects, we'll see issues.

Reply:
A pull request 🤔. So you commit to this proposal even though it has not been decided which design to choose?

Given your comment, does that mean that maybe you would be in favour of #165 after all?

Reply:
> So you commit to this proposal

No, we're experimenting with the various options to provide feedback. And good that we are, because that's what led to my comment above 🙂